functime/feature_extraction/tsfresh.py

```diff
@@ -49,12 +49,12 @@ def absolute_maximum(x: TIME_SERIES_T) -> FLOAT_EXPR:
     float | Expr
     """
     if isinstance(x, pl.Series):
-        return np.max(np.abs([x.min(), x.max()]))
```

Comment:
I wrote it this way using NumPy for the following reason:

Calling abs() on the entire series will trigger a temporary allocation of a new series as long as the original x. By only computing x.min() and x.max() and taking the absolute value, you avoid this allocation. On larger data, this improves performance by a lot.

You can verify mathematically the max of the absolute value must be either |x.min()| or |x.max()|, so creating an entire series by x.abs() is unnecessary.

The old implementation is better.